src/sentry/reprocessing2.py

```diff
@@ -240,8 +240,8 @@ def reprocess_event(project_id: int, event_id: str, start_time: float) -> None:
     )
 
 
-def get_original_group_id(event: Event) -> int | None:
-    return get_path(event.data, "contexts", "reprocessing", "original_issue_id")
+def get_original_group_id(event: Event) -> int:
+    return event.data["contexts"]["reprocessing"]["original_issue_id"]
```

Comment:
this would crash later on when accessing `group_id` as `None` -- so I believe this always has to be present